Tests/UnitTests/Purchasing/Purchases/PurchasesConfiguringTests.swift

```diff
@@ -158,6 +158,29 @@ class PurchasesConfiguringTests: BasePurchasesTests {
         expect(purchases.appUserID) == Self.appUserID
     }
 
+    func testStaticUserIdSringLogsMessage() {
+        let logger = TestLogHandler()
+
+        _ = Purchases.configure(
+            with: .init(withAPIKey: "")
+                .with(appUserID: "Static string")
+        )
+
+        logger.verifyMessageWasLogged(Strings.identity.logging_in_with_static_string, level: .warn)
+    }
+
+    func testUserIdSringDoesNotLogMessage() {
+        let appUserID = "user ID"
+        let logger = TestLogHandler()
+
+        _ = Purchases.configure(
+            with: .init(withAPIKey: "")
+                .with(appUserID: appUserID)
+        )
+
+        logger.verifyMessageWasNotLogged(Strings.identity.logging_in_with_static_string, level: .warn)
```

Comment:
I recommend removing the level here to make sure it's not logged at _any_ level:

```suggestion
        logger.verifyMessageWasNotLogged(Strings.identity.logging_in_with_static_string)
```